src/dewrapper/dewrapper.c

```diff
@@ -152,6 +152,10 @@ static k4a_result_t depth_engine_start_helper(dewrapper_context_t *dewrapper,
         if (deresult != K4A_DEPTH_ENGINE_RESULT_SUCCEEDED)
         {
             LOG_ERROR("Depth engine create and initialize failed with error code: %d.", deresult);
+            if (deresult == K4A_DEPTH_ENGINE_RESULT_FATAL_ERROR_GPU_OPENGL_CONTEXT)
+            {
+                LOG_ERROR("OpenGL context creation failed. You could try updating your graphics drivers.", 0);
```

Comment:
do we need to be more specific on the opengl version minimum requirement in this message?

Reply:
I thought about that but then realized if we change the version in depth engine dll then this message will be out of sync. So I don't know, should we define the version here?

Reply:
I'll add the version.

Reply:
How about: "OpenGL context creation failed. Requires OpenGLv4.4 or above. You could try updating your graphics drivers.

---
In reply to: [309440703](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/pull/566#discussion_r309440703) [](ancestors = 309440703)